frontend/packages/dev-console/src/components/pipelines/resource-overview/ResourceLinkList.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { ResourceLink } from '@console/internal/components/utils';
+import './ResourceLinkList.scss';
+
+type ResourceLinkListProps = {
+  namespace: string;
+  kind: string;
+  title: string;
+  links: string[];
+};
+const ResourceLinkList: React.FC<ResourceLinkListProps> = ({
+  links = [],
+  kind,
+  namespace,
+  title,
+}) => {
+  if (links.length === 0) {
+    return null;
+  }
+  return (
+    <div className="odc-resource-link-list">
+      <dl>
+        <dt>{title}</dt>
+        {links?.map((name) => {
+          return (
+            <React.Fragment key={name}>
+              <dd>
+                <ResourceLink kind={kind} name={name} namespace={namespace} title={name} inline />
+              </dd>
+            </React.Fragment>
```

Comment:
```suggestion
              <dd key={name}>
                <ResourceLink kind={kind} name={name} namespace={namespace} title={name} inline />
              </dd>
```

Fragments are there to support multiple "root" children. Recommend you read the [Fragments page on the React docs](https://reactjs.org/docs/fragments.html) as this is not the first time I have mentioned a fragment issue.

Reply:
I was not sure why added this :facepalm: , clearly we dont need this, will remove it.